frontend/src/components/AddDatasetDrawer/AddDatasetDrawer.tsx

```diff
@@ -1,14 +1,56 @@
+import RaisedButton from 'material-ui/RaisedButton'
 import * as React from 'react'
-import { withStore } from '../../services/store'
+import { Dataset } from '../../constants/datatypes'
+import { Store, withStore } from '../../services/store'
 import { BackLink } from '../Link/Link'
 import { ProvidersUplodader } from '../Uploader/ProvidersUplodader'
 import { ServiceAreasUploader } from '../Uploader/ServiceAreasUplodader'
+import './AddDatasetDrawer.css'
 
-export let AddDatasetDrawer = withStore('selectedDataset')(({}) =>
+export let AddDatasetDrawer = withStore('selectedDataset')(({ }) =>
   <div className='AddDatasetDrawer'>
-    <BackLink/>
+    <BackLink />
     <h2 className='Secondary'>Upload your data to explore</h2>
-    <ServiceAreasUploader/>
-    <ProvidersUplodader/>
+    <p className='MediumWeight Muted'>
+      To analyze the accessibility of your own set of providers, facilities, or social services, you
+      will need to upload two seperate CSV files: <br />
+
+      - list of service areas (County and/or ZIPCode columns). <br />
+
+      - list of addresses for providers or services.
+    </p>
+    <ServiceAreasUploader />
+    <ProvidersUplodader />
+    <AnalyzerButton />
```

Comment:
Can we have a bit more of a gap between the upload things and the "go" button? I feel like it could do with a bit more emphasis in some way.

Reply:
Could you help me put in lower and centerer it?